package.json

```diff
@@ -56,6 +62,7 @@
   },
   "proxy": "http://localhost:8000",
   "devDependencies": {
+    "@size-limit/preset-app": "^4.5.0",
```

Comment:
You still need to specify size-limit as a dependency here.

Reply:
Ah, so that's why it wasn't working. I can't read.